### PR TITLE
Modify manually managed  key for apt repositories

### PR DESCRIPTION
**Pull request by Kartik494**
Fixes: #38053 

This PR modify path from `/usr` to `/etc` while manually managed key for apt repositories

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -172,6 +172,9 @@ For more information on version skews, see:
    sudo apt-get install -y kubelet kubeadm kubectl
    sudo apt-mark hold kubelet kubeadm kubectl
    ```
+{{< note >}}
+In releases older than Debian 12 and Ubuntu 22.04, /etc/apt/keyrings does not exist by default.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:176]: (nit)
```suggestion
In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.
You can create this directory if you need to, making it world-readable but writeable only by admins.
```

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -135,6 +135,9 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    sudo apt-get update
    sudo apt-get install -y kubectl
    ```
+{{< note >}}
+In releases older than Debian 12 and Ubuntu 22.04, /etc/apt/keyrings does not exist by default.
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:139]: (nit)
```suggestion
In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.
You can create this directory if you need to, making it world-readable but writeable only by admins.
```

**Review by sftim — COMMENTED** [review]
/lgtm
/sig cluster-lifecycle

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md (hunk 1 above)
Comment (Kartik494) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: Done

File: content/en/docs/tasks/tools/install-kubectl-linux.md (hunk 2 above)
Comment (Kartik494) [reply content/en/docs/tasks/tools/install-kubectl-linux.md]: Addressed the changes. PTAL

**Pull request merged**